yelp_beans/logic/metrics.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+from yelp_beans.logic.meeting_spec import get_specs_for_current_week
+from yelp_beans.logic.meeting_spec import get_users_from_spec
+from yelp_beans.models import MeetingSubscription
+from yelp_beans.models import User
+
+
+def get_subscribed_users():
+    users = User.query().fetch()
+    subscriptions = MeetingSubscription.query().fetch()
+
+    metrics = defaultdict(set)
+    for subscription in subscriptions:
+        metrics[subscription.key.urlsafe()] = []
```

Comment:
Since `metrics` is a `defaultdict`, why do we need this initialization code here?